website/docs/reference/resource-configs/batch_size.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "batch_size"
+id: "batch-size"
+sidebar_label: "batch_size"
+resource_types: [models]
+description: "dbt uses `batch_size` to detrmine how large batches are when running a microbatch incremental model."
+datatype: hour | day | month | year
+---
+
+Available in dbt Cloud Versionless and dbt Core v1.9 and higher.
```

Comment:
Would it be ok if we version the whole page? Any issues with that?

Reply:
No issue with that. The only reason I didn't was because I directly copied and then edited the `event_time.md` page